src/app/models/user.go

```diff
@@ -23,12 +23,12 @@ type (
 	}
 
 	UserJson struct {
-		Id      int       `json:"id"`
-		Name    string    `json:"name"`
-		Email   *string   `json:"email"`
-		Image   *string   `json:"image"`
-		Created time.Time `json:"created"`
-		Updated time.Time `json:"updated"`
+		Id      int        `json:"id"`
```

Comment:
__[golint]__
* [golint] struct field Id should be ID

<!-- Comment by [SideCI](https://sideci.com) -->